### PR TITLE
Update capstone_project_overview.md

### PR DESCRIPTION
**Pull request by case-eee**
@neight-allen here's my first pass at updating my sections

* updated the overview section
* updated the roles section
* updated the structure of the week

needs updated:
* how students submit their project proposal's is currently just a markdown, i'm going to make a file for each student to update and submit a PR

### CODE REVIEW

File: module4/capstone_project_overview.md [hunk 1]
```diff
@@ -17,42 +17,26 @@ Learning Goals
...
 
-After the demos, each team will meet with their Technical Lead who will evaluate their progress via the rubric below.
+At the beginning of each week, each student will set their sprint (aka what they want to accomplish that week). Throughout the week, each student should be tagging their Technical Lead in PR's if they have questions or want feedback on any code they've written. 
 
-During the first day of each sprint, each Technical Lead will meet with the Product Owner of each team to walk through the next set of stories. The Product Owner is responsible for brainstorming what these stories are with their team. The Technical Lead is responsible for helping guide the Product Owner and estimate how long each story will take. 
+Halfway through the project, each student will spend 2 minutes demoing their project to the entire class. Students will also demo their projects at the end of the project in week six.
```
Comment (neight-allen) [inline module4/capstone_project_overview.md]: We don't currently have time on the calendar to demo in week 6. Should we just do it on Wednesday morning along with their open source demos?

Comment (case-eee) [inline module4/capstone_project_overview.md]: i think that sounds great 👍

**Comment by neight-allen** [comment]
Should we mention the retro and self assessment?

**Comment by case-eee** [comment]
definitely @neight-allen - i'll add that!

**Comment by neight-allen** [comment]
@case-eee Not sure this was the right way to do things, but I just put my changes on the same PR. Could you take a look?

File: module4/capstone_project_overview.md [hunk 2]
```diff
@@ -17,73 +17,60 @@ Learning Goals
 Overview
 ------------
 
-The Capstone Project will last two weeks and will consist of students working in teams of 3-4 students. Teams will be formed based on interest in an existing Turing internal projects or the cohort's existing personal projects listed [here](https://gist.github.com/BobGu/8e1aa089c9fafc99c0c557dfcbbf9080).
+The Capstone Project will last two weeks and will consist of students meaningful, publishable work of their choosing. An instructor will be assigned to each student and act as the Technical Lead.
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: love the "publishable" add 👍

File: module4/capstone_project_overview.md [hunk 3]
```diff
@@ -17,73 +17,60 @@ Learning Goals
...
-
-	1.  Project Manager
-	2.  Product Owner
+Technical Expectations
+---------------------
 
+We would like to give you a fair amount of freedom on this project. We want you to bulid something that you're excited about, and that you're proud of. These are a few requirements you should consider when pitching and building your project:
 
-*   An instructor will be assigned to each group and act as the Technical Lead
+*   You build something usable. Your should be comfortable publishing your project by the time it's due in week 6. A spike or exploration into a new concept without a clear deliverable won't be accepted.
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: typo here - "Your" should be "You"

File: module4/capstone_project_overview.md [hunk 4]
```diff
@@ -17,73 +17,60 @@ Learning Goals
...
+*   You must apply at least two new techniques. New is something from the 4th module (OOJS, Security, Environments, React), or something extra-curricular. Something not taught in the Turing Back-end engineering program.
+	-   Since accessibility is already in the rubric, it doesn't count as a new topic
```
Comment (case-eee) [inline module4/capstone_project_overview.md]: maybe add something like ", but it is a requirement"

**Comment by case-eee** [comment]
@neight-allen - added some comments and took your feedback as well. let me know what you think!

**Comment by neight-allen** [comment]
@case-eee Fixes made. @tmikeschu, could you give this a once over before we merge it?

File: module4/capstone_project_overview.md [hunk 5]
```diff
@@ -17,73 +17,68 @@ Learning Goals
...
+*   3: Student is able to set and update expectations so that there are no surprises on the last day of the sprint
+*   2: Student does not have agreed upon stories completed at the end of the sprint, but has a plan to get them done
+*   1: Student does not have agreed upon stories completed, and has no plan to complete them
 
 ### 3. Implementation Quality
 
-*   4: Project demonstrates exceptionally well factored code.
-*   3: Project exhibits maintainable well divided code. Developers are able to speak to architecture and implementation decisions.
+*   4: Project demonstrates exceptionally well tested and maintainable code.
+*   3: Project exhibits tested, maintainable, well divided code. Developers are able to speak to architecture and implementation decisions.
```
Comment (tmikeschu) [inline module4/capstone_project_overview.md]: I assume 4 encapsulates the architecture/implementation speakability?

**Pull request merged**